www/client/index.html

```diff
@@ -7,11 +7,10 @@
     <li><a href="/server/">Server&nbsp;Features</a></li>
     <li><a href="/client/">Client&nbsp;Features</a></li>
     <li><a href="/map/">Map&nbsp;Features</a></li>
-    <li><a href="/howto/">How&nbsp;to&nbsp;Map</a></li>
```

Comment:
The https://ddnet.org/howto/ link seems to be a redirect configured outside of this repository. That should also be adjusted to redirect to https://ddnet.org/mapping/ when this PR is merged.